l10n_it_delivery_note/models/stock_delivery_note.py

```diff
@@ -274,6 +274,18 @@ def _domain_weight_uom(self):
     sale_count = fields.Integer(compute="_compute_sales")
     sales_transport_check = fields.Boolean(compute="_compute_sales", default=True)
 
+    company_currency_id = fields.Many2one(
+        "res.currency",
+        "Company Currency",
+        related="company_id.currency_id",
+    )
+    amount_total = fields.Monetary(
+        "Total Amount",
+        compute="_compute_amount_total",
+        currency_field="company_currency_id",
+        store=True,
+    )
+
```

Comment:
**issue**: I prezzi delle DN lines potrebbero essere espressi in valute diverse da quella della company.